Tmain/omit-long-patterns-etags.d/stdout-expected.txt

```diff
@@ -0,0 +1,5 @@
+
+input.sh,349
+                                                                                                func96(func961,0
```

Comment:
Oh, sorry. My code truncates lines from their tail. 
I think it should do from head. fixup commit is pushed. Could you look at the commit again?